README.md

```diff
@@ -31,7 +33,22 @@ globParent('path/**/*'); // 'path'
 globParent('path/foo/bar.js'); // 'path/foo'
 globParent('path/foo/'); // 'path/foo'
 globParent('path/foo'); // 'path' (see issue #3 for details)
+```
+
+## Escaping
+
+The following characters have special significance in glob patterns and must be escaped if you want them to be treated as regular path characters:
 
+- `?` (question mark)
+- `*` (star)
+- `[` (opening bracket)
+- `]` (closing bracket)
+
+**Example**
+
+```js
+globParent('foo/[bar]/') // 'foo'
+globParent('foo/\\[bar]/') // 'foo/\\[bar]'
```

Comment:
> why would we ever remove backslashes from escaped characters?

Because the output of glob-parent should be an actual path, free from any form of glob-ness and ready to pass into things like `fs.readdir`